### PR TITLE
Hero section

### PR DESCRIPTION
**Pull request by lx1922**
### Ticket
[Ticket Link](https://trello.com/c/eg3hLjAV/6-home-page-hero-box)

### What is changed / added
- Added Card Component
- Made a hero section using card component
-- this is without the arrow in the button

### Screenshots
- image

### CODE REVIEW

**Review by nigow — COMMENTED** [review]
could you rebase to the master branch?

**Comment by lx1922** [comment]
> could you rebase to the master branch?

Yes I'll be doing that today.

**Comment by lx1922** [comment]
Additionally, this is without the arrow on learn more button.

File: pages/_app.tsx [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
 
 function MyApp({ Component, pageProps }: AppProps) {
-  return <Component {...pageProps} />
+  useEffect(() => {
+    const jssStyles = document.querySelector('#jss-server-side');
+    if (jssStyles?.parentElement) {
+      jssStyles.parentElement.removeChild(jssStyles);
+    }
+  }, []);
+  return <Component {...pageProps} />;
```
Comment (TheFatPanda97) [inline pages/_app.tsx:13]: I think this is left over from material UI, definitely want to remove this.

File: tsconfig.json [hunk 2]
```diff
@@ -12,7 +12,10 @@
     "moduleResolution": "node",
     "resolveJsonModule": true,
     "isolatedModules": true,
-    "jsx": "preserve"
+    "jsx": "preserve",
+    "noImplicitAny": true,
+    "noImplicitThis": true,
+    "strictNullChecks": true
```
Comment (TheFatPanda97) [inline tsconfig.json:18]: why did we add these configs?

File: styles/theme.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    error: {
+      main: red.A400,
+    },
+    background: {
+      default: '#fff',
+    },
+  },
+});
+
+export default theme;
```
Comment (TheFatPanda97) [inline styles/theme.js:22]: Need to be removed, left over from material UI as well

**Review by TheFatPanda97 — CHANGES_REQUESTED** [review]
The component looks good, but we definitely want to merge master, there are some things that look outdated (like the material UI code).

**Comment by lx1922** [comment]
> The component looks good, but we definitely want to merge master, there are some things that look outdated (like the material UI code).

Yes! @nigow suggested a rebase and I was gonna do it tonight!

File: pages/_app.tsx (hunk 1 above)
Comment (lx1922) [reply pages/_app.tsx:13]: removed! :)

File: tsconfig.json (hunk 2 above)
Comment (lx1922) [reply tsconfig.json:18]: Not sure I dont remember adding these!

File: styles/theme.js (hunk 3 above)
Comment (lx1922) [reply styles/theme.js]: done!

**Comment by TheFatPanda97** [comment]
[Home Page - Hero Box](https://trello.com/c/eg3hLjAV/6-home-page-hero-box)

**Review by nigow — APPROVED** [review]

**Review by NerajM — APPROVED** [review]

**Comment by lx1922** [comment]
> The component looks good, but we definitely want to merge master, there are some things that look outdated (like the material UI code).

Done, now!

**Review by TheFatPanda97 — APPROVED** [review]
Ready to go!

**Pull request merged**